src/main/java/Duke.java

```diff
@@ -1,10 +1,258 @@
+        System.out.println(OUTPUT_DIVIDER);
+    }
+
+    public static boolean isNumeric(String str) {
+        try {
+            Integer.parseInt(str);
+            return true;
+        } catch (NumberFormatException e) {
+            return false;
+        }
+    }
+
+    public static void markAndPrintTaskDone(Task[] tasks, int taskCount, String doneTask) {
+        String[] doneSentence = doneTask.split(" ");
+
+        // Checks if done command entered does not follow the correct format of "done {task ID}".
+        if (doneSentence.length != 2) {
+            System.out.println(OUTPUT_DIVIDER);
+            System.out.println(MESSAGE_COMMAND_WRONG_FORMAT);
+            System.out.println(MESSAGE_DONE_CORRECT_FORMAT);
+            System.out.println(OUTPUT_DIVIDER);
+            return;
+        }
+
+        // Checks if the task ID entered is numeric.
+        if (!isNumeric(doneSentence[1])) {
+            System.out.println(OUTPUT_DIVIDER);
+            System.out.println(MESSAGE_COMMAND_WRONG_FORMAT);
+            System.out.println(MESSAGE_NUMERIC_TASK_ID);
+            System.out.println(OUTPUT_DIVIDER);
+            return;
+        }
+
+        int taskToMarkDone = Integer.parseInt(doneSentence[1]);
+        // Makes sure that the task being mark done is in the task list.
+        if (taskToMarkDone > taskCount || taskToMarkDone <= 0) {
+            System.out.println(OUTPUT_DIVIDER);
+            System.out.println(MESSAGE_TASK_NOT_IN_LIST);
+            System.out.println(OUTPUT_DIVIDER);
+        } else {
+            tasks[taskToMarkDone - 1].setDone(true);
+            System.out.println(OUTPUT_DIVIDER);
+            System.out.println(MESSAGE_TASK_MARKED_DONE);
+            System.out.println("   " + tasks[taskToMarkDone - 1].toString());
+            System.out.println(OUTPUT_DIVIDER);
+        }
+    }
+
+    public static boolean canAddAndPrintTodo(Task[] tasks, int taskCount, String addedTodo) {
+        String[] splitTodo = addedTodo.split("\\s+", 2);
+
+        // Makes sure that the todo entered is of the format "todo {description}"
+        if(splitTodo.length != 2) {
+            System.out.println(OUTPUT_DIVIDER);
+            System.out.println(MESSAGE_COMMAND_WRONG_FORMAT);
+            System.out.println(MESSAGE_CORRECT_TODO_FORMAT);
+            System.out.println(OUTPUT_DIVIDER);
+            return false;
+        }
+
+        String todoDescription = splitTodo[1];
+        Todo newTodo = new Todo(todoDescription);
+        tasks[taskCount] = newTodo;
+        System.out.println(OUTPUT_DIVIDER);
+        System.out.println(MESSAGE_TASK_ADDED);
+        System.out.println("   " + newTodo.toString());
+        System.out.println(" You now have " + (taskCount + 1) + " task(s) in the list.");
+        System.out.println(OUTPUT_DIVIDER);
+        return true;
+    }
+
+    public static boolean canAddAndPrintDeadline(Task[] tasks, int taskCount, String addedDeadline) {
```

Comment:
Similar to the "canAddAndPrintToDo" method, should the method name describe it performing 2 distinct tasks and how it is appropriate for the boolean return type (Similar cases are also found elsewhere).